From de88826db1bcc7fb51dcc060df47b9853eeb6573 Mon Sep 17 00:00:00 2001
From: Suresh Gupta <suresh.gupta@freescale.com>
Date: Sun, 14 Apr 2013 20:25:46 +0530
Subject: [PATCH 431/547] fsl/usb: Turn on cache snooping hardware for 64-bit
 Kernel

Turn on cache snooping hardware for 64-bit Kernel

Signed-off-by: Suresh Gupta <suresh.gupta@freescale.com>
Change-Id: I1f59eb6a53fd420cda87873bec0ec82094a022de
Reviewed-on: http://git.am.freescale.net:8181/1320
Reviewed-by: Mehresh Ramneek-B31383 <ramneek.mehresh@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
[Original patch taken from QorIQ-SDK-V1.4-SOURCE-20130625-yocto.iso]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 drivers/usb/gadget/fsl_udc_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/gadget/fsl_udc_core.c b/drivers/usb/gadget/fsl_udc_core.c
index c005dc7..9ac6b97 100644
--- a/drivers/usb/gadget/fsl_udc_core.c
+++ b/drivers/usb/gadget/fsl_udc_core.c
@@ -335,7 +335,7 @@ static int dr_controller_setup(struct fsl_udc *udc)
 	}
 #endif
 
-#if defined(CONFIG_PPC32) && !defined(CONFIG_NOT_COHERENT_CACHE)
+#if !defined(CONFIG_NOT_COHERENT_CACHE)
 	/* Turn on cache snooping hardware, since some PowerPC platforms
 	 * wholly rely on hardware to deal with cache coherent. */
 
-- 
1.8.4.93.g57e4c17

